include/dlaf/blas/tile.h

```diff
@@ -114,12 +114,12 @@ DLAF_MAKE_GPUBLAS_SYHE_OP(Her2k, r2k);
 
 DLAF_MAKE_GPUBLAS_SYHE_OP(Herk, rk);
 
-#ifdef DLAF_WITH_HIP
+#if defined(__clang__) && defined(DLAF_WITH_HIP)
```

Comment:
Do these warnings not pop up when compiling host code? If yes, there should probably be an else branch for GCC to ignore the same warnings.